packages/cli/src/commands/env.js

```diff
@@ -0,0 +1,64 @@
+const { Command, flags } = require("@oclif/command");
+const fs = require("fs");
+const { loadRcFile } = require("../common/spashiprc-loader");
+const { assign, get, isEmpty, omit } = require("lodash");
+const { config } = require("../common/config/index");
+const commonFlags = require("../common/flags");
+const os = require("os");
+//process.env["NODE_TLS_REJECT_UNAUTHORIZED"] = 0;
```

Comment:
Remove commented code

Reply:
OR add a readable comment explicitly mentionioning the purpose

Reply:
will change it

Reply:
Resolved